app/src/routes/home.tsx

```diff
@@ -57,31 +63,21 @@ const NewUser = (user) => {
 
 };
 
-const GetDishes = (user) =>{
-  const [dishesUsed, setDishesUsed] = useState([]);
-
-  const location = useLocation();
-  let sessionToken = location.state
-  useEffect(()=>{
-    axios.get('http://localhost:8080/api/transactions', {headers:{"x-api-key":"test","session-token":sessionToken}})
-    .then(function (response) {
-      setDishesUsed(response.data.transactions)
-      console.log("data:",response.data.transactions)
-    })
-    .catch(function (error) {
-      console.log(error);
-    });
-  },[])
 
-  let dishes = DishAPI.getUserActiveDishes(user.uid);
+const GetDishes = (dishesUsed) =>{
+  let dishesInUse = 0;
   return (  
     <div id="dishes" style={{marginTop: '24px'}}>
         <div className="d-flex justify-content-between">
           <p className="sub-header-3">My Dishes</p>
-          {/* <p> {data.transactions.length} </p> */}
-          <p className="details-2 mt-1">{dishesUsed.length} in use</p>
+          {dishesUsed.map(dish => {
+            if (dish.returned.timestamp == null){
+              dishesInUse = dishesInUse + 1
```

Comment:
It looks like you want the number of dishes that are still checked out? I wouldn't use a map for this, but a filter: 

```
const checkedOutDishes = dishesUsed.filter(dish => dish.returned.timestamp === null).length
```